### PR TITLE
Profile Tab: Fix improper navigation on closing create bounty tab

### PR DESCRIPTION
**Pull request by ChetanKarwa**
## Issue ticket number and link
https://github.com/stakwork/sphinx-tribes-frontend/issues/78

Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome 
- [x] I have provided a recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Can you post a video?

**Comment by ecurrencyhodler** [comment]
LGTM. Let's get a code review.

File: src/pages/people/tabs/Wanted.tsx [hunk 1]
```diff
@@ -107,12 +107,8 @@ export const Wanted = observer(() => {
               }}
               widget={'wanted'}
               onSucces={() => {
-                history.goBack();
```
Comment (kevkevinpal) [inline src/pages/people/tabs/Wanted.tsx:110]: can you post a video on the `onSucces` seems like we only saw the `onGoBack` cause I think the `onSucces` takes us to the bounties page and we might want to go back in this scenario

**Comment by ecurrencyhodler** [comment]
@elraphty actually this is a relevant issue as it is part of the onboarding process for a brand new provider. If you sign in with an lnurl user you'll be able to recreate this issue. I was able to do it.

**Comment by kevkevinpal** [comment]
i rebased to master we should try and avoid the commits that say the below next time

`Merge branch 'master' into ...`

**Comment by ecurrencyhodler** [comment]
Tested and paid!

**Comment by ecurrencyhodler** [comment]
tested and paid!

**Pull request merged**